4 new additions - 2 old and 2 new projects

NavenduShekhar opened the pull request:
I am adding 4 projects

NavenduShekhar committed:
Changes

NavenduShekhar committed:
Changes

NavenduShekhar committed:
Changes

KrishnaKumarTiwari reviewed (commented):
@NavenduShekhar rename

* The folder name should be the lower case with dash as a word separator 
* Read should be something like README.md
https://github.com/ML-AI-Community/ml-ai/blob/master/fake-news-detection/README.md
* Remove Titanic folder

NavenduShekhar committed:
changes

NavenduShekhar committed:
Create README.md

NavenduShekhar committed:
Create README.md

NavenduShekhar commented:
Sir, I have deleted those 2 projects, renamed the projects as you have mentioned and created Readme files in the respective folders. Please have a look again.
